Textile/Completions/Inline Symbols.sublime-completions

```diff
@@ -0,0 +1,62 @@
+{
+    "scope": "text.html.textile",
+    "completions":
+    [
+        {
+            "trigger": "trademark",
+            "contents": "(tm)",
+            "annotation": "Unicode symbol",
+            "kind": ["snippet", "s", "Snippet"],
+            "details": ""
```

Comment:
Not that important, but you could put the actual Unicode symbol in the details tag.